src/labels/CustomLabels.labels

```diff
@@ -5101,21 +5101,45 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>pmtWizardMsgNoOppFound</shortDescription>
         <value>No opportunity found.</value>
     </labels>
+    <labels>
+        <fullName>pmtWizardInvalidRecords</fullName>
+        <categories>Payment Wizard</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>pmtWizardInvalidRecords</shortDescription>
+        <value>NOTE: This Opportunity has payment record(s) that are both Paid and Written Off. You must deselect one or both checkboxes.</value>
+    </labels>
+    <labels>
+        <fullName>pmtWizardScheduleNoAmount</fullName>
+        <categories>Payment Wizard</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
All labels have to be marked as protected.

Reply:
@force2b thanks - I keep forgetting that as I copy and paste but should we just have a WI to convert them all to protected in one go?